tests/js/server/recovery/view-search-alias-populate-fail.js

```diff
@@ -40,22 +41,38 @@ function runSetup () {
 
   var meta = { indexes: [ { index: i1.name, collection: c.name() } ] };
   db._view('UnitTestsRecoveryView').properties(meta);
-
   var tx = {
     collections: {
       write: ['UnitTestsRecoveryDummy']
     },
     action: function() {
+      const db = require('@arangodb').db;
+      const internal = require('internal');
       var c = db.UnitTestsRecoveryDummy;
       for (let i = 0; i < 10000; i++) {
         c.save({ a: "foo_" + i, b: "bar_" + i, c: i });
       }
-
-      c.save({ name: 'crashme' }, true);
+      c.save({ name: 'dontcrashme' }, true);
+      print('crash?')
```

Comment:
crash caused `internal.debugTerminate('crashing server');`
I don't understand why do you do these changes

Reply:
this very test doesn't crash the server using this method - hence I wanted it not to use this name.
Since else the valued reader could think that this should crash the server - but it doesn't.